setup.cfg

```diff
@@ -0,0 +1,21 @@
+[flake8]
+# D203  1 blank line required before class docstring
+# H201  no 'except:' at least use 'except Exception:'
+# H302  import only modules
+# H405  multi line docstring summary not separated with an empty line
+ignore = D203,H201,H302,H405
+# H106: Don’t put vim configuration in source files
+# H203: Use assertIs(Not)None to check for None
+enable-extensions = H106,H203
+max-complexity = 50
+max-line-length = 120
+exclude =.env,.git,doc,*lib/python*,build,dist,*.egg
+show-source = True
+builtins = _
+
+[pylint]
+disable = C0103,C0111,F0010
```

Comment:
Put comments with description what are you disabling (like in 2-5 lines)

Reply:
done